docs/source/conf.py

```diff
@@ -0,0 +1,217 @@
+""" 
+
+# -- Options for HTML output ----------------------------------------------
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
+#
+html_theme = 'sphinx_scylladb_theme'
+html_theme_path = ["../.."]
+html_style = ''
+
+# Theme options are theme-specific and customize the look and feel of a theme
+# further.  For a list of options available for each theme, see the
+# documentation.
+#
+
+html_theme_options = {
+    'header_links': [
+    ('Scylla Rust Driver', 'https://scylladb.github.io/rust-driver/'),
+    ('Scylla Cloud', 'https://docs.scylladb.com/scylla-cloud/'),
+    ('Scylla University', 'https://university.scylladb.com/'),
+    ('ScyllaDB Home', 'https://www.scylladb.com/')],
+    'site_description': 'Scylla Driver for Rust.',
+    'github_issues_repository': 'scylladb/scylla-rust-driver',
+    'show_sidebar_index': 'false',
+    'hide_banner': 'true',
+    'default_branch': 'stable'
+}
+
+extlinks = {
+    'manager': ('/operating-scylla/manager/%s/',''),
+    'manager_lst': ('/operating-scylla/manager/2.0/%s/',''),
+    'monitor': ('/operating-scylla/monitoring/%s/',''),
+    'monitor_lst': ('/operating-scylla/monitoring/3.1/%s/','')
+}
+
+# If not None, a 'Last updated on:' timestamp is inserted at every page
+# bottom, using the given strftime format.
+# The empty string is equivalent to '%b %d, %Y'.
+#
+html_last_updated_fmt = '%d %B %Y'
+
+# Custom sidebar templates, maps document names to template names.
+#
+html_sidebars = {'**': ['side-nav.html']}
+
+# Output file base name for HTML help builder.
+htmlhelp_basename = 'ScyllaDocumentationdoc'
+
+# URL which points to the root of the HTML documentation. 
+html_baseurl = 'https://sphinx-theme.scylladb.com'
+
+# Dictionary of values to pass into the template engine’s context for all pages
+html_context = {'html_baseurl': html_baseurl}
+
+# -- Options for not found extension -------------------------------------------
+
+# Template used to render the 404.html generated by this extension.
+notfound_template =  '404.html'
+
+# Prefix added to all the URLs generated in the 404 page.
+notfound_urls_prefix = ''
+
+# -- Options for redirect extension ---------------------------------------
+
+# Read a YAML dictionary of redirections and generate an HTML file for each
+redirects_file = "_utils/redirections.yaml"
+
+# -- Options for multiversion extension ----------------------------------
+
+# Whitelist pattern for tags (set to None to ignore all tags)
+TAGS = ['0.1.0']
+smv_tag_whitelist = multiversion_regex_builder(TAGS)
+# Whitelist pattern for branches (set to None to ignore all branches)
+BRANCHES = ['release_0_1_0']
+smv_branch_whitelist = multiversion_regex_builder(BRANCHES)
+# Defines which version is considered to be the latest stable version.
+# Must be listed in smv_tag_whitelist or smv_branch_whitelist.
+smv_latest_version = '0.1.0'
```

Comment:
Until the tag is not created, documentation for the tag 0.1.0 will not be built. For this reason, it might be best to list `smv_latest_version = '0.1.0'` for now.